librz/bin/p/bin_elf.inc

```diff
@@ -1385,6 +1386,7 @@ static RzBinReloc *reloc_convert(ELFOBJ *bin, RzBinElfReloc *rel, ut64 GOT) {
 		case RZ_X86_64_GOTPCREL: ADD(64, GOT - P);
 		case RZ_X86_64_COPY: ADD(64, 0); // copy symbol at runtime
 		case RZ_X86_64_IRELATIVE: r->is_ifunc = true; SET(64);
+		case RZ_X86_64_TPOFF64: break;
```

Comment:
@XVilka not sure if these are correct.

Reply:
Please extract relocation fixes into a separate PR. We should check how they are defined in binutils as well.

Reply:
Reverting my changes on this file :+1: